<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsultarDialog</class>
 <widget class="QWidget" name="ConsultarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonVoltar">
     <property name="text">
      <string>VOLTAR </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Localidade:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxLocalidade">
       <item>
        <property name="text">
         <string>Jericoacoara</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canoa Quebrada</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cumbuco</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Numero de diarias:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxDiarias">
       <property name="minimumSize">
        <size>
         <width>73</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Data de check-in:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditCheckIn"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tipo de quarto:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2TipoQuarto">
       <item>
        <property name="text">
         <string>Solteiro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Duplo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Triplo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Qu√°druplo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Casal</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonConsultar">
     <property name="text">
      <string>CONSULTAR</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
